website/versioned_docs/version-28.0/UpgradingToJest28.md

```diff
@@ -140,6 +140,13 @@ The constructor of [test environment](Configuration.md#testenvironment-string) c
 +     const config = projectConfig;
 ```
 
+In addition, test environments `"jest-environment-node"` is now exporting it's environment with a default export, instead of simply exporting the class directly:
+
+```diff
+- const NodeEnvironment = require('jest-environment-node');
++ const NodeEnvironment = require('jest-environment-node').default;
```

Comment:
I think `{TestEnvironment}` is better. Same applies to the jsdom env, fwiw

Reply:
Sure, updated it ✅